CHANGELOG.md

```diff
@@ -6,9 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [0.0.21] - 2022-08-17
-### Fixed
-- HTTP 403 error for URLs returned by `MessagingClient.getFileUrl` (#37).
```

Comment:
Don't delete the entry for `0.0.21`.

Reply:
doh! I was sure I had copied and pasted it... sorry!